--- ast_utils.py	2022-12-08 11:29:25.577944185 +0800
+++ ast_utils_new.py	2022-12-08 10:54:08.492424086 +0800
@@ -648,8 +648,9 @@
     if not force_rebuild and os.path.exists(file_path):
         wrapped_index = _load_index(file_path)
         md5 = file_scanner.files_mtime_md5()
-        if (wrapped_index[VERSION_KEY] == __version__
-                and wrapped_index[MD5_KEY] == md5):
+        #if (wrapped_index[VERSION_KEY] == __version__
+        #        and wrapped_index[MD5_KEY] == md5):
+        if wrapped_index[VERSION_KEY] == __version__:
             index = wrapped_index
 
     if index is None:
